src/main/webapp/app/courses/exercises/exercise-list.html

```diff
@@ -45,6 +47,13 @@
                 <i class="fa fa-circle-o-notch fa-spin" ng-show="$ctrl.loading[exercise.id.toString()]"></i>
             </button>
             <button class="btn btn-primary btn-sm btn-block"
+                    ng-switch-when="active"
+                    ng-click="$ctrl.start(exercise)"
+                    ng-disabled="$ctrl.loading[exercise.id.toString()]">
+                <span ng-hide="$ctrl.loading[exercise.id.toString()]"><i class="fa fa-play-circle fa-fw"></i>&nbsp;Resume exercise</span>
```

Comment:
Should this not be "Resume quiz"?

Reply:
I would either keep "Resume Exercise" or also change the "Start Exercise" to "Start Quiz" (if the exercise is a quiz) to be consistent.

Reply:
Then please change it to "Start Quiz" as well